bar/bootstrap.sh

```diff
@@ -1,9 +1,19 @@
 #!/usr/bin/bash
```

Comment:
Highly suggest to add some spaces between logical groups of code.
this make your code more readable and easily to maintain in the feature.